Source/WebKit/WebProcess/WebCoreSupport/WebChromeClient.cpp

```diff
@@ -166,10 +166,6 @@ void WebChromeClient::didRemoveMenuItemElement(HTMLMenuItemElement& element)
     m_page.didRemoveMenuItemElement(element);
 }
 
-inline WebChromeClient::~WebChromeClient()
```

Comment:
Generally, non inlined dtor is a good practice to reduce the number of headers to include.
If you don't have a performance problem you have to inline, I'd like to keep it non inlined.

Reply:
`WebChromeClient::~WebChromeClient() = default;` in the cpp sounds good to me.

Reply:
I understand, and there are multiple ways to fix it. The destructor doesn't need to be inlined here.

Reply:
I think marking methods with `inline` in a cpp file has no actual effect. So, this dtor wasn't inlined, I think.

Reply:
https://github.com/WebKit/WebKit/commit/0df6f3c9e212ad674dcd048424981066c87d950a added the dtor in the cpp file.
I think it's the author's intention. Marking `inline` was just a mistake.

Reply:
I like Chris's suggestion.  "inline" should be removed.